modules/cnvkit/main.nf

```diff
@@ -40,13 +40,16 @@ process CNVKIT {
         target_args = "--targets $targetfile"
     }
 
+    normal_args = normalbam ? "--normal $normalbam" : ""
+
+    fasta_args = fasta ? "--fasta $fasta" : ""
     """
     cnvkit.py \\
         batch \\
         $tumourbam \\
```

Comment:
I do prefer the tumor over tumour spelling
But it's mainly me being picky

Reply:
Will do once I fix this current failure 😄 🇺🇸🇺🇸🇺🇸🇺🇸🇺🇸🇺🇸